usersexercise/src/main/java/com/Controllers/UsersController.java

```diff
@@ -0,0 +1,72 @@
+package com.Controllers;
+
+import com.Model.Users;
+import com.Services.UserService;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.beans.BeanUtils;
+import com.Model.Response.UserResponse;
+import com.dto.UserDto;
+import com.Model.Request.UserRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("users")
+public class UsersController {
+
+    private final UserService userservice;
+
+    public UsersController(UserService userservice){
+        this.userservice = userservice;
+    }
+
+    @GetMapping(path = "/allUsers")
```

Comment:
paths for REST apis should be collection/{individual}/property, and differentiate between them using the HTTP protocol verbs. So this would just be GET /users with no extra pathing